python_modules/dagster/dagster/_generate/templates/CODE_LOCATION_NAME_PLACEHOLDER/pyproject.toml.tmpl

```diff
@@ -1,7 +1,25 @@
+[project]
+name = "{{ code_location_name }}"
+version = "0.1.0"
+description = "Add your description here"
+readme = "README.md"
+requires-python = ">=3.8,<3.12"
+dependencies = [
+    "dagster",
+    "dagster-cloud",
+    "pandas",
```

Comment:
is there a reason to add `pandas` by default? folks might not want it as a dependency

Reply:
Should the dependencies match this file?
https://github.com/dagster-io/dagster/blob/master/python_modules/dagster/dagster/_generate/templates/REPO_NAME_PLACEHOLDER/setup.py.tmpl

I added pandas in the issue because the quickstart scaffold has a dependency on pandas.

Reply:
No pandas would match here. 
I only added pandas because it was an explicit dependency in the dagster quickstart in Getting Started. Maybe pandas could be removed from the quickstart too?

Reply:
Yea, I think we remove pandas, and use `uv add pandas` if we need it during a tutorial.